<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDialog</class>
 <widget class="QMainWindow" name="GameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢乐连连看</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameSet">
      <property name="styleSheet">
       <string notr="true">.QFrame{
	background-image:url(:/source/images/res/theme.jpg);
	background-repeat:no-repeat;
}
.QPushButton
{
	border-radius:5px;
	height:25px;
	background-color:white;
	border:1px solid blac
}
.QPushButton{
	color:black;
	background-color:rgb(202,174,126);
}

.QPushButton:pressed{
	color:white;
}
.QPushButton:hover{
	background-color:white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>669</x>
         <y>30</y>
         <width>101</width>
         <height>521</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnStart">
          <property name="text">
           <string>开始游戏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPause">
          <property name="text">
           <string>暂停游戏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTips">
          <property name="text">
           <string>提示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReset">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTool">
          <property name="text">
           <string>道具</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAuto">
          <property name="text">
           <string>自动解题</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReview">
          <property name="text">
           <string>过程回放</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnHelp">
          <property name="text">
           <string>帮助</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetting">
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetDraw" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>671</width>
         <height>541</height>
        </rect>
       </property>
      </widget>
      <widget class="QProgressBar" name="barLeftTime">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>550</y>
         <width>561</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
      <widget class="QLabel" name="labLeftTime">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>550</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:16px;</string>
       </property>
       <property name="text">
        <string>剩余时间</string>
       </property>
      </widget>
      <widget class="QLabel" name="LabTime">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>550</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:16px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelGrade">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>532</y>
         <width>61</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelGradeInfo">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>529</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前积分：</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
